src/components/Calculator.js

```diff
@@ -0,0 +1,101 @@
+/* eslint-disable react/prefer-stateless-function */
+import React, { Component } from 'react';
+
+export default class Calculator extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
+  render() {
+    const firstRow = ['AC', '+/-', '%'];
+    const secondRow = [9, 8, 7];
+    const thirdRow = [6, 5, 4];
+    const fourthRow = [3, 2, 2];
```

Comment:
- [ ] Please fix the typo mistake I see double **2** in the **fourthRow**.

![jsjs](https://user-images.githubusercontent.com/61562626/133248125-1f7443f3-8303-40e6-9ce4-477dd320f81f.jpeg)

Reply:
done